RELEASE-NOTES.rst

```diff
@@ -1,36 +1,20 @@
 ===========================
- Invenio-Classifier v0.1.0
+ Invenio-Classifier v1.0.0
 ===========================
 
-Invenio-Classifier v0.1.0 was released on August 19, 2015.
+Incompatible changes
+--------------------
 
-About
------
+- Changes module to be compatible with Invenio 3.
 
-Invenio module for record classification.
+Bug fixes
+---------
 
-*This is an experimental development preview release.*
+- Fixes a crash when trying to discover a taxonomy when
+  CLASSIFIER_WORKDIR is set to None.
+- Updates minimum dependencies of Invenio packages to newer versions.
+- Removes a bug in bibclassify_keyword_analyzer.py. If a combination
+  is found via a synonym or regexp it is no longer thrown away just
+  because the components of the combination are not found in the text.
+- Adds missing `invenio_base` dependency.
 
-What's new
-----------
-
-- Initial public release.
-
-Installation
-------------
-
-   $ pip install invenio-classifier==0.1.0
-
-Documentation
--------------
-
-   http://invenio-classifier.readthedocs.org/en/v0.1.0
-
-Happy hacking and thanks for flying Invenio-Classifier.
-
-| Invenio Development Team
```

Comment:
This is a good point. Wasn't entirely sure about what to put it. But I guess I shall.